apps/openchallenges/app/src/app/app.component.scss

```diff
@@ -0,0 +1,9 @@
+mat-sidenav-container,
+mat-sidenav-content,
+mat-sidenav {
```

Comment:
Are these existing class that we are modifying? Should we define our own classes instead that overwrite selected properties?